src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -68,6 +275,14 @@ public boolean isUnregisteredStudent() {
         return account.googleId == null || student != null && !student.isRegistered();
     }
     
+    public static void createListForNationalities(String nationality) {
```

Comment:
You can use varargs here `createListForNationalities(String... nationalities)` so that multiple entries can be added to the list in one method call.